packages/template/src/components/team-switcher.tsx

```
@@ -76,7 +77,7 @@ function Inner<AllowNull extends boolean>(props: TeamSwitcherProps<AllowNull>) {
 
   const navigate = app.useNavigate();
   const project = app.useProject();
-  const rawTeams = user?.useTeams();
+  const rawTeams = props.teams ?? user?.useTeams();
```

Biome (2.3.13)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)